python.d/redis.chart.py

```diff
@@ -18,10 +18,15 @@
 #             'unix_socket': None
 #          }}
 
-ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'slaves', 'persistence',
+ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'eviction', 'slaves', 'persistence',
          'bgsave_now', 'bgsave_health']
 
 CHARTS = {
+    'eviction':{
+        'options': [None, 'Evicted keys', 'keys', 'evicted', 'redis.evictios', 'line'],
```

Comment:
sorry for not being clear. `keys/s` is ok when algorithm is `incremental` and wrong if `absolute`. Netdata applies interpolation to find the exact value at second-boundaries only for `incremental` values, so `/s` is misleading for `absolute`.

Now units should be `keys/s` 😄

Reply:
https://github.com/firehol/netdata/pull/1758#issuecomment-279153641